lisa/main.py

```diff
@@ -16,50 +16,48 @@
 
 
 @retry(FileExistsError, tries=10, delay=0)
-def create_run_path(root_path: Path) -> Path:
+def layout_runtime_folder():
```

Comment:
The original method is for retry on collision log paths. If it needs to get code out of main path, create another method.

Reply:
That part (test path and raise exception) is still in this function with retry.

Reply:
The retry is like a `try..catch...` statement, its scope should be as small as possible to prevent unexpected behavior. The code in retry should be able to idempotent, so it can be rerun. (this place meets this requirement).

Reply:
Will do it later. But it's your private branch, not sure if I have permission to push changes.